README.md

```diff
@@ -1 +1,33 @@
-# cl-snapshot-streaming
\ No newline at end of file
+# Snapshot Streaming
+
+## Building
+
+In the root directory of [tessellation](https://github.com/cngo-github/tessellation) run
+
+```
+git checkout v1.9.1
```

Comment:
As well here we should checkout and build the version from `project/Dependencies.scala`

Reply:
As above, I think we should avoid sophisticated temporary solutions.